keyboards/gami_studio/lex60/readme.md

```diff
@@ -0,0 +1,12 @@
+# Fox Lab Leaf60
+
+A 60% with split space bars made by Gami Studio.
+
+Keyboard Maintainer: [Gami Studio](https://github.com/GamiStudio)
+Hardware Supported: Lex60 powered by the ATmega32U4
+
+Make example for this keyboard (after setting up your build environment):
+
+    make gami_studio/Lex60:default # postfix with `:dfu` to load the firmware into the board
```

Comment:
The `make` command is case-sensitive. Also, QMK has a `:flash` target that will work as long as the MCU and Bootloader definitions in `rules.mk` are accurate.

```suggestion
    make gami_studio/lex60:default # postfix with `:flash` to load the firmware into the board
```